port/common/omrcuda.cpp

```diff
@@ -995,29 +995,29 @@ const J9CudaLibraryDescriptor runtimeLibraries[] = {
 /*
  * Include forward-compatible support for runtime libraries.
  */
-#if 8000 <= CUDA_VERSION
+#if CUDA_VERSION <= 8000
```

Comment:
My understanding is that this code is building a list of all possible libraries that could be loaded based on the CUDA_VERSION macro from cuda.h.  If the version is 8000 create an entry for all of the known libraries <= to 8000.  Then code later on will try to open the libraries in descending order so that it opens the most `recent` library?